<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExamInfoPage</class>
 <widget class="QWidget" name="ExamInfoPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Exam Information</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="basicInfoGroup">
     <property name="title">
      <string>Basic Information</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="titleLabel">
        <property name="text">
         <string>Exam Title:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="titleEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="subjectLabel">
        <property name="text">
         <string>Subject:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="subjectEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="classLabel">
        <property name="text">
         <string>Class/Grade:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="classEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="termLabel">
        <property name="text">
         <string>Term/Semester:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="termEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="detailsGroup">
     <property name="title">
      <string>Exam Details</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="durationLabel">
        <property name="text">
         <string>Duration:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="durationEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="totalMarksLabel">
        <property name="text">
         <string>Total Marks:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="totalMarksSpin"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="passMarksLabel">
        <property name="text">
         <string>Pass Marks:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="passMarksSpin"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="dateLabel">
        <property name="text">
         <string>Exam Date:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDateEdit" name="examDateEdit"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="orientationLabel">
        <property name="text">
         <string>Landscape:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="landscapeCheckBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="nextButton">
       <property name="text">
        <string>Next &gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>